Makefile

```diff
@@ -2,7 +2,34 @@ XCODE_USER_TEMPLATES_DIR=/Applications/Xcode.app/Contents/Developer/Library/Xcod
 TEMPLATE_NAME=VIPER
 TEMPLATES_DIR=Example/Templates/VIPER
 
+EXISTS_FASTLANE = $(shell command -v fastlane 2> /dev/null)
+
 install_templates:
 	mkdir -p $(XCODE_USER_TEMPLATES_DIR)
 	rm -fR $(XCODE_USER_TEMPLATES_DIR)/$(TEMPLATE_NAME)
-	cp -R $(TEMPLATES_DIR) $(XCODE_USER_TEMPLATES_DIR)
\ No newline at end of file
+	cp -R $(TEMPLATES_DIR) $(XCODE_USER_TEMPLATES_DIR)
+
+install_env:
+ifeq "${EXISTS_FASTLANE}" ""
+	@echo Installing fastlane
+	sudo gem install fastlane --no-document
+endif		
+	@echo "All dependencies was installed"
+
+build_dapp:
+	fastlane build scheme:DApp
+
+build_wallet:
+	fastlane build scheme:WalletApp
+
```

Comment:
If I understand correctly it will build apps with the every CI check?
If so I think it's a little bit overhead for the resources and time. This check would make more sense to me when merging to `master` i.e.

Reply:
Our tests is running in parallel, so CI time depends on longes test case. Now it is Integration tests. I can switch off this case if it make sense for you